addons/google_recaptcha/__manifest__.py

```diff
@@ -16,6 +16,11 @@
         'web.assets_frontend': [
             'google_recaptcha/static/src/scss/recaptcha.scss',
             'google_recaptcha/static/src/js/recaptcha.js',
+            # TODO remove next line in master, it was kept here as part of a
+            # stable fix in stable but is unused in the current codebase.
+            'google_recaptcha/static/src/xml/recaptcha.xml',
+        ],
+        'web.assets_backend': [
```

Comment:
Mmh, still was not what I had in mind but I guess that in the spirit of not lazy loading the XML anymore, that would be the right fix for stable. We'll review the XML mess in master.

I'll add more information to the commit message myself though.